### PR TITLE
Feature: GitHub get started section

### PR DESCRIPTION
**Pull request by natapg**
First GitHub Landing page section

ticket : https://fpjs.myjetbrains.com/youtrack/issue/WEB-115

### CODE REVIEW

**Review by tan-90 — APPROVED** [review]
Everything looks good to me, I like how the page looks.

File: src/components/githubPage/GetStartedSection/GetStartedSection.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import Container from '../../common/Container'
+import ActionableCard, { Card } from '../../ActionableCard/ActionableCard'
+import { useMainBackgroundImage } from '../../../hooks/useBackgroundImage'
+import { URL } from '../../../constants/content'
+
+import styles from './GetStartedSection.module.scss'
+
+export default function GetStartedSection() {
+  const { mainBackground } = useMainBackgroundImage()
+  const cards = [
```
Comment (Lisenish) [inline src/components/githubPage/GetStartedSection/GetStartedSection.tsx:12]: Minor: Looks like we can create this outside of the component (so we don't recreate them on each re-render

File: src/components/ActionableCard/ActionableCard.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  variant?: 'primary' | 'outline' | 'clear' | 'faded'
+}
+
+export interface ActionableCardProps {
+  cards: Card[]
+  perRow?: 2 | 3 | 4
+  className?: string
+}
+
+export default function ActionableCard({ cards, className, perRow = 2 }: ActionableCardProps) {
```
Comment (Lisenish) [inline src/components/ActionableCard/ActionableCard.tsx:22]: Minor: The name is in a singular from but this components actually renders collection of card, so I would name it `ActionableCards`

**Review by Lisenish — APPROVED** [review]
@natapg Approved in advance but please take a look at comments (merge by yourself when you fixed them)

**Pull request merged**